enclave/src/test_polkadex_gateway.rs

```diff
@@ -80,6 +81,7 @@ pub fn create_mock_gateway() -> OpenfinexPolkaDexGateway<OpenFinexApiMock> {
 }
 
 pub fn initialize_storage() {
+    assert!(create_in_memory_channel_storage().is_ok());
```

Comment:
What are you trying to do here?  What to do when`create_in_memory_channel_storage`  is Err?

Reply:
It throws an error - but since it's a test that what we want I guess?

Reply:
Did not see it was in a test.  Resolved